config/locales/en.yml

```diff
@@ -134,6 +151,29 @@ en:
     related_content_title: Related content
     not_applicable: Not applicable
   estimate_flow:
+    benefits_more:
+      caption: Your client's finances
+      heading:
+        text: You have added %{count} benefits
```

Comment:
You'll need to pluralize `benefit` appropriately so that it doesn't say "1 benefits", won't you?